providers/mlx5/mlx5dv.h

```diff
@@ -1444,7 +1444,7 @@ struct mlx5dv_dr_flow_meter_attr {
 	uint8_t                 active;
 	uint8_t                 reg_c_index;
 	size_t			flow_meter_parameter_sz;
-	__be64			flow_meter_parameter[];
+	void			*flow_meter_parameter;
```

Comment:
So what actually gets put in here? We don't have a pyverbs test for this?

Can you snip an example from the user?

Reply:
The usage as 'void *' can be seen in below links from DPDK upstream git.

Create meter:
http://git.dpdk.org/dpdk/tree/drivers/net/mlx5/mlx5_glue.c #l776

Prepare meter:
http://git.dpdk.org/dpdk/tree/drivers/net/mlx5/mlx5_flow_meter.c #l60